src/Test/contlinear.jl

```diff
@@ -1135,6 +1153,11 @@ function linear10test(model::MOI.ModelLike, config::TestConfig)
             @test MOI.get(model, MOI.DualStatus()) == MOI.FEASIBLE_POINT
             @test MOI.get(model, MOI.ConstraintDual(), c) ≈ -1 atol=atol rtol=rtol
         end
+        if config.basis
+            @test (MOI.get(model, MOI.VariableBasisStatus(), x) == MOI.BASIC ||
```

Comment:
Is this because there are multiple optimal bases? Please add a comment explaining.